### ISSUE TITLE
Keyboard navigation in file list does not work when rows are hidden

### ISSUE THREAD

**Issue by jonashertzman** [jonashertzman/FileSearch3#6]
.NET bug, not sure if this can be fixed https://stackoverflow.com/questions/12650343/some-rows-are-collapsed-in-datagrid-i-am-getting-issue-in-keyboard-navigation
